src/site/content/en/blog/stories-component/index.md

```diff
@@ -25,36 +36,54 @@ Let's explore and get inspired by this minimal way to layout an interactive Stor
   </video>
 </figure>
 
-### Introduction
-Stories are commonly found as a mobile only, tap centric pattern for navigating multiple subscriptions. For example: on Instagram, users open a friend's story and go through the pictures in it. They generally do this many friends at a time. By tapping on the right side of the device, users move forward through "their stories," which are their friends and their friends' stories.
+If you would prefer a hands-on demonstration of building this Stories component yourself,
+check out the [Stories component codelab](TODO).
 
-**Today's is mobile web focused and supports keyboard navigation.** We'll also be testing cross platform too, sharing the differences we end up with and why it's okay and awesome.  
-
-{% Banner 'neutral' %}This is the written companion for the following YouTube video.{% endBanner %}
+If you prefer video, here's a YouTube version of this post:
 
 {% YouTube '-oyeaIirVC0' %}
 
-## The Web Platform
-This component was able to come together so smoothly because of a few critical features of the web that made it all possible. Let's cover some of them!
+## Overview
+
+Two popular examples of the Stories UX are Snapchat Stories and Instagram Stories.
+In general UX terms, Stories are usually a mobile-only, tap-centric pattern for navigating
+multiple subscriptions. For example, on Instagram, users open a friend's story
+and go through the pictures in it. They generally do this many friends at a
+time. By tapping on the right side of the device, a user skips ahead to that friend's
+next story. By swiping right, a user skips ahead to a different friend's story.
+A Story component is fairly similar to a carousel, but allows navigating a
+multidimensional array as opposed to an array. It's as if there's a carousel inside
+each carousel. 🤯 
+
+## Picking the right tools for the job {: #features }
+
+All in all I found this component pretty easy to build, thanks to a few
+critical new web platform features. Let's cover some of them!
+
+### CSS Grid {: #grid }
```

Comment:
I moved the Grid stuff to the top because I think it's easier to grok the overall layout if you start from a top-down kind of approach